### PR TITLE
Implement Accept Alert tests for WPT wdspec tests

### PR DESCRIPTION
**Pull request by alchiem**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None


<h2>Failing Jobs</h2>
<ul>

<li>firefox:nightly</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18564)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/275668426)

**Comment by alchiem** [comment]
Testing webdriver accept alert spec.
http://www.w3.org/TR/webdriver/#accept-alert

File: webdriver/tests/user_prompts/accept_alert.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+def test_accept_confirm(session):
+    # 18.2 step 3
+    session.execute_script("window.confirm(\"Hello\");")
+    response = accept_alert(session)
+    assert_success(response)
+
+
+def test_accept_prompt(session):
+    # 18.2 step 3
+    session.execute_script("window.prompt(\"Enter Your Name: \", \"Federer\");")
```
Comment (andreastt) [inline webdriver/tests/user_prompts/accept_alert.py]: Accepting a prompt will return an empty string, whereas dismissing it returns null. You should set this variable on the window global and then verify that the prompt is actually getting accepted.

**Review by andreastt — CHANGES_REQUESTED** [review]
This looks mostly good. Just one change needed. Please fix up the commits as well.

File: webdriver/tests/user_prompts/accept_alert.py (hunk 1 above)
Comment (alchiem) [reply webdriver/tests/user_prompts/accept_alert.py]: fixed

**Comment by alchiem** [comment]
Hi @andreastt,

What do you mean by fix up the commits as well?

Thanks,
-Alex

**Comment by andreastt** [comment]
> What do you mean by fix up the commits as well?

Sorry, I should’ve explained this in more detail.

I want you to squash your commits together into one with a good commit message. This branch currently has five commits: the original “Implement Accept Alert tests” followed by four fixup commits. You can fold the latter four commits into the first commit.

**Review by andreastt — APPROVED** [review]

**Comment by andreastt** [comment]
@alchiem Disregard that. I was able to do it for you through the GitHub UI.

**Comment by alchiem** [comment]
Hey @andreastt, 

So I added the same changes from dismiss alert to this branch where I moved the assert_success before the assert on the result of the dialog. Do I make another branch for that since you merged in the changes already?

**Comment by andreastt** [comment]
@alchiem Ah yes, I missed that here. If you would file another PR to fix that, I would be most obliged.

**Pull request merged**